docs/examples/general/data_loading/external_input.ipynb

```diff
@@ -4,10 +4,10 @@
    "cell_type": "markdown",
```

Comment:
<p>For this example: Two spaces between "For" and "this"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA/DALI/pull/2480/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>

Reply:
<p>Hmm, I don't see it in the source. Maybe some rendering issue? I'll make sure that it is ok in the final build when I have it.</p>

Reply:
<blockquote>create an ExternalInputGpuIterator so it returns</blockquote><p>Semantics: we've already created one.</p><p>Suggested alternative: modify ExternalInputGpuIterator so that it returns</p><p> </p>

Reply:
<p>Before we created <code>ExternalInputGpuIterator</code>  and now we create <code>ExternalInputGpuIterator</code> . </p><p> </p><p>I changed it to:  "For this example, we create the <code>ExternalInputGpuIterator</code> that returns data on the GPU." to make it clear that this is something new.</p>